samples/test/update-intent.test.js

```diff
@@ -0,0 +1,77 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const {assert} = require('chai');
+const {describe, before, it, after} = require('mocha');
+const execSync = require('child_process').execSync;
+const uuid = require('uuid');
+const dialogflow = require('@google-cloud/dialogflow-cx');
+const exec = cmd => execSync(cmd, {encoding: 'utf8'});
+const projectId =
```

Comment:
we don't use env variables anymore, instead we use [this](https://github.com/googleapis/nodejs-dlp/blob/b4a612d15508e7b0f513a0b75a4d2123759a60b5/samples/system-test/deid.test.js#L35), up to line 40